docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -624,6 +624,10 @@ How to **investigate**:
         ./tools/markblocks/markblocks -backend gcs -gcs.bucket-name <bucket> -mark no-compact -tenant <tenant-id> -details "Result block exceeds symbol table maximum size" <block-1> <block-2>...
         ```
     - Further reading: [Compaction algorithm]({{< relref "../../references/architecture/components/compactor#compaction-algorithm" >}}).
+  - Compactor network disk unresponsive:
+    - **How to detect**: A telltale clue is many cores of sustained kernel-mode CPU usage by the compactor process. Check the metric `rate(container_cpu_system_seconds_total{pod="<pod>"}[$__rate_interval])` for the affected pod.
+    - **What it means**: The compactor process has frozen because it is blocked on kernel-mode flushes to an unresponsive network block storage device.
```

Comment:
```suggestion
    - **What it means**: The compactor process has frozen because it's blocked on kernel-mode flushes to an unresponsive network block storage device.
```